puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-katello-test.yml

```diff
@@ -1,45 +1,80 @@
+---
 - job-template:
-    name: 'foreman-katello-{version}-test'
-    project-type: pipeline
+    name: foreman-katello-{version}-{distro}-{action}-test
+    project-type: 'pipeline'
     sandbox: true
+    parameters:
+      - string:
+          name: expected_version
+          default: 'UNKNOWN'
```

Comment:
`UNKNOWN` will cause the tests to fail. It should be empty if you want to skip it:
https://github.com/theforeman/forklift/blob/0f03bebf38c6941aecb50923585c9f7515aa3fed/bats/fb-test-foreman.bats#L6-L10

In the Foreman pipelines this is ok because it's called with it:
https://github.com/theforeman/theforeman-rel-eng/blob/2d82d085b4a49a723fad33cdeb7ee90c599dd5d1/release_pipeline#L5

We can set it to `2.1.0-develop` on nightly.